operator/pkg/ciliumendpointslice/manager.go

```diff
@@ -45,6 +47,7 @@ type operations interface {
 	// External APIs to Insert/Remove CEP in local dataStore
 	InsertCEPInCache(cep *cilium_v2.CoreCiliumEndpoint, ns string) string
 	RemoveCEPFromCache(cepName string, baseDelay time.Duration)
+	CompressCESsOnFCFSStartUp()
```

Comment:
Why do we need to have it in 2 places?

Reply:
This is just added to the operations interface definition, so that it can be called elsewhere.
The manager is exposed in the controller through operations interface.
https://github.com/cilium/cilium/blob/master/operator/pkg/ciliumendpointslice/endpointslice.go#L66